pkg/kubelet/metrics/collectors/volume_stats.go

```diff
@@ -61,6 +61,12 @@ var (
 		[]string{"namespace", "persistentvolumeclaim"}, nil,
 		metrics.ALPHA, "",
 	)
+
+	volumeStatsHealthAbnormalDesc = metrics.NewDesc(
+		metrics.BuildFQName("", kubeletmetrics.KubeletSubsystem, kubeletmetrics.VolumeStatsHealthStatusKey),
+		"Volume health status. The count is either 1 or 0",
```

Comment:
Instead of specifying that the count is either 1 or 0, it might be better to mention that 1 means that the PVC is healthy and 0 unhealthy.

Reply:
I think after we added label `volume_health_status: "abnormal"` to this metrics. We don't need to explain more in here. Right?

Reply:
I think it is still helpful to add more comments here.